### PR TITLE
Preset: Add "requireSpacesInsideParentheses" to jquery preset

### PR DESCRIPTION
**Pull request by markelog**

### CODE REVIEW

**Comment by markelog** [comment]
@sfrisk, @arschmitz, @scottgonzalez, @mzgol since you voted №2 in https://github.com/jquery/contribute.jquery.org/pull/104, new code will look like that, still good?

File: test/data/options/preset/jquery.js [hunk 1]
```diff
@@ -85,8 +85,8 @@ jQuery.extend = jQuery.fn.extend = function() {
       }
 
       // requireOperatorBeforeLineBreak
-      if ( deep && copy && ( jQuery.isPlainObject(copy) ||
-          (copyIsArray = jQuery.isArray(copy)) ) ) {
+      if ( deep && copy && ( jQuery.isPlainObject( copy ) ||
+          ( copyIsArray = jQuery.isArray( copy ) ) ) ) {
```
Comment (markelog) [inline test/data/options/preset/jquery.js]: Spaces are awesome! Since everyone says so :-).

/cc @dmethvin

**Comment by mikesherov** [comment]
Yes, it's still good. LGTM.

**Comment by scottgonzalez** [comment]
Looks correct. Thanks.

**Comment by arschmitz** [comment]
LGTM

**Pull request closed without merging**